hallux/backends/factory.py

```diff
@@ -69,13 +69,12 @@ def _create_backend(
 
         # Add backward compatibility for openai and openai.azure types
         # TODO: remove this in the future versions
-        if backend_type in ["openai", "openai.azure"]:
+        if backend_type.lower() in ["openai", "openai.azure"]:
             backend_type = "litellm"
             settings["type"] = "litellm"
             logger.warning(f"'openai' and 'openai.azure' backends are deprecated. Please use 'litellm' instead.")
```

Comment:
Add replacement fields or use a normal string instead of an f-string.
```suggestion
            logger.warning("'openai' and 'openai.azure' backends are deprecated. Please use 'litellm' instead.") 
```

Reply:
good job 👍